tests/test_profiling_utils.py

```diff
@@ -47,6 +51,8 @@ def test_env_var_enable():
 
 # Create nested ranges and verify the code runs without error.
 def test_nested_range_push_pop():
+    if not habitat_sim.cuda_enabled:
```

Comment:
Why do these tests require that habitat-sim was compiled with CUDA?  That is different than there being CUDA capable devices on the system